service/dap/server.go

```diff
@@ -1417,6 +1550,23 @@ func (s *Server) onCancelRequest(request *dap.CancelRequest) {
 	s.sendNotYetImplementedErrorResponse(request.Request)
 }
 
+// sendErrorResponseWithURL sends and error response to be visible to the user with an error related URL.
+func (s *Server) sendErrorResponseWithURL(request dap.Request, id int, summary, details string, url string, urlLabel string) {
```

Comment:
combine the argument types `(request dap.Request, id int, summary, details, url, urlLabel string)`